src/main/resources/global_deploy.conf

```diff
@@ -15,12 +15,9 @@ project = {
   version = null
 }
 deploy = {
-  // Hostname to push to. Required.
+  // Hostname to push to. Required if 'replicas' not set, forbidden if 'replicas' is set.
   host = null
-  // Directory on the remote host to push to. Required.
-  directory = "/local/deploy/"${?project.name}
-  // Start / stop script to run after the push is complete. Required.
-  startup_script = "bin/"${?project.name}".sh"
+  // User data for the host above. Required if 'host' is set.
```

Comment:
Is the intent that this will be used for all hosts? It should be - think should normally never change.